sdk/eventhub/event-hubs/samples/eventProcessor.ts

```diff
@@ -47,12 +41,13 @@ class SimplePartitionProcessor {
     console.log(`Encountered an error: ${error.message}`);
   }
 
-  async initialize() {
-    console.log(`Started processing`);
+  async initialize(partitionContext: PartitionContext) {
+    console.log(`Started processing partition: ${partitionContext.partitionId}`);
   }
 
-  async close() {
-    console.log(`Stopped processing`);
+  async close(reason: CloseReason, partitionContext: PartitionContext) {
+    console.log(`Stopped processing for reason ${reason}`);
```

Comment:
Will this print out the reason in string or number?

Reply:
This prints out the reason as a string.

Reply:
That said, I think we should probably change the `CloseReason` enum to a union of string literals. Then it's easy to get auto-completion, and clear that it's a string.